test/resources/controls/aws_parallelcluster_slurm/slurm_dependencies_spec.rb

```diff
@@ -12,6 +12,29 @@
 control 'slurm_dependencies_installed' do
   title "Slurm dependencies are installed"
 
+  def lua_devel_package
+    if os.debian?
+      # Retrieves version of installed lua lib
+      lua_version_output = command("dpkg-query -W  -f='${Package} ${Version}\n' liblua*").stdout
```

Comment:
See comment above.

Reply:
Also, which version are we really testing against? What if for whatever reason multiple versions of liblua-dev get installed?

Reply:
As for the other comments, after discussing the options we agreed to fix the lua version.